demo/auth/passport.js

```diff
@@ -0,0 +1,47 @@
+var $fh = require('fh-js-sdk');
+var logger = require('@raincatcher/logger').logger;
+var ConsoleLogger = require('@raincatcher/logger').ConsoleLogger;
+var setLogger = require('@raincatcher/logger').setLogger;
+
+$fh.init({}, function() {
```

Comment:
Improper use of fh.init. This code doesn't make any sense.
We should wrap only specific calls to `$fh.getCloudURL()` not entire api.